### PR TITLE
Bugfix/main menu and table adjustment

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+# 3.0.1
```
Comment (proand) [inline CHANGELOG.md:1]: `# Unreleased`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+> Oct 15, 2021
+
+* :bug: **Bugfix** FHI main menu breakpoint fix and removed border below table heading
+
 # 3.0.0
```
Comment (proand) [inline CHANGELOG.md:7]: `## 3.0.0`

File: CHANGELOG.md (hunk 1 above)
Comment (yleaxeman) [reply CHANGELOG.md:1]: It was this that made me believe I could add the new version immediately, and create a release just after merging this PR:
Screenshot 2021-10-15 at 10 57 29

Comment (proand) [reply CHANGELOG.md:1]: Hmm... forgot that one... I guess yes, but not certain if it's a good practice or not. It's less overhead to create a release branch right away, so my gut feeling is that it's it a good idea :)

**Pull request merged**